conans/cli/commands/user.py

```diff
@@ -0,0 +1,110 @@
+    else:
+        info = {"results": {"remote1": {"user": "someuser1"}}}
+    return info
+
+
+# no user associated with remote yet
+@conan_subcommand()
+def user_add(*args, conan_api, parser, subparser):
+    """
+    Add user authentication for a remote.
+    """
+    subparser.add_argument("remote", help="Remote name")
+    # conan user add command should ask for the name and password in case none of them are
+    # passed as arguments. If just --name, ask for password. If --password is passed ask for
+    # username
+    subparser.add_argument("-n", "--name", action=OnceArgument,
+                           help="User password. Use double quotes if password with "
+                                "spacing, and escape quotes if existing. If empty, the "
+                                "password is requested interactively (not exposed)")
+    subparser.add_argument("-p", "--password", action=OnceArgument,
+                           help="User password. Use double quotes if password with "
+                                "spacing, and escape quotes if existing. If empty, the "
+                                "password is requested interactively (not exposed)")
+    subparser.add_argument("-f", "--force", action='store_true', default=False,
+                           help="Force addition, will update if existing.")
+    args = parser.parse_args(*args)
+    return {}
+
+
+@conan_subcommand()
+def user_remove(*args, conan_api, parser, subparser):
+    """
+    Remove associated user from remote.
+    """
+    subparser.add_argument("remote",
+                           help="Remote name. Accepts 'fnmatch' style wildcards. "
+                                "To remove the user for all remotes use: conan remote remove '*'")
+    args = parser.parse_args(*args)
+    return {}
+
+
+@conan_subcommand()
+def user_update(*args, conan_api, parser, subparser):
+    """
+    Update the current user for a remote. If not 'user' and 'password' are passed it will ask
+    for them interactively.
+    """
+    subparser.add_argument("remote", help="Remote name")
+    # changing the name of the user forces a password change, it will be requested interactively
+    # if not passed as an argument
+
+    subparser.add_argument("-n", "--name", action=OnceArgument,
+                           help="Name of the new user. If no name is specified the "
+                                "command will update the current user.")
+    subparser.add_argument("-p", "--password", nargs="?", const="", type=str, action=OnceArgument,
+                           help="Update user password. Use double quotes if password "
+                                "with spacing, and escape quotes if existing. If "
+                                "empty, the password is requested interactively "
+                                "(not exposed)")
+    args = parser.parse_args(*args)
+    return {}
+
+
+@conan_command(group="Misc commands")
```

Comment:
Remove the "commands" from the group, let the groups be called "Consumer", "Creator", "Misc", and the "commands can be added in output only if necessary.